src/psyclone/tests/domain/lfric/lfric_domain_kernels_test.py

```diff
@@ -297,8 +297,8 @@ def test_psy_gen_domain_kernel(dist_mem, tmpdir, fortran_writer):
     else:
         expected = "      ! call our kernels\n"
     assert (expected + "      !\n"
-            "      !\n"
-            "      call testkern_domain_code(nlayers, ncell_2d_no_halos, b, "
+            "      do dummy = 1, 1, 1\n"
```

Comment:
As queried earlier, can't we just remove this loop altogether?

Reply:
I now got rid of this loops as in the original.